internal/transform/schema.go

```diff
@@ -149,7 +149,7 @@ type TrustlineOutput struct {
 	AssetIssuer        string      `json:"asset_issuer"`
 	AssetType          int32       `json:"asset_type"`
 	Balance            float64     `json:"balance"`
-	TrustlineLimit     int64       `json:"trust_line_limit"`
+	TrustlineLimit     int64       `json:"trustline_limit"`
```

Comment:
The `TrustlineLimit` column in the `trust_lines` table is still `trust_line_limit`. Is there a reason that we're changing the schema to `trustline_limit` here? Just want to make sure they continue matching.

Reply:
You're right that this shouldn't be included. Removed.